keps/sig-node/2535-ensure-secret-pulled-images/README.md

```diff
@@ -179,10 +178,27 @@ or expect preloaded images since boot.
 
 ## Design Details
 
-Kubelet will track, in memory, a hash map for the credentials that were successfully used to pull an image. The hash map
-will not be persisted to disk, in alpha. For alpha explicitly, we will not reuse or add other state manager concepts to kubelet.
+Kubelet will track, in memory, a hash map for the credentials that were successfully used to pull an image. It has been decided that the hash map will be persisted to disk, in alpha.
 
-See PR for detailed design / behavior documentation.
+See `/var/lib/kubelet/image_manager_state` in [kubernetes/kubernetes#114847](https://github.com/kubernetes/kubernetes/pull/114847)
+
+> ```
+> {
+>   "images": {
+>     "sha256:eb6cbbefef909d52f4b2b29f8972bbb6d86fc9dba6528e65aad4f119ce469f7a": {
+>       "authHash": {
+>         "115b8808c3e7f073": {
```

Comment:
This should use SHA 256 since this is a security boundary.

Reply:
ok

Reply:
made comments to move to sha256 vs hash .. agree esp. since it's being stored (even if temporarily)